model/registry/db.yaml

```
@@ -245,19 +245,14 @@ groups:
           the database name to be used is the more specific layer (e.g. Oracle schema name).
         examples: [ 'customers', 'main' ]
         tag: db-generic
-      - id: operation
+      - id: operation.name
         type: string
         stability: experimental
         brief: >
-          The name of the operation being executed, e.g. the [MongoDB command name](https://docs.mongodb.com/manual/reference/command/#database-operations)
-          such as `findAndModify`, or the SQL keyword.
+          The name of the operation being executed.
```

```suggestion
          The name of the operation or command being executed.
```

Maybe? In strict SQL terminology, it's usually a "[command](https://www.w3schools.com/sql/sql_syntax.asp)" that's expected here (for example `SELECT`, or `INSERT`).

I also wonder if that would make the footnote obsolete, which seems to be a duplication of what's in the examples for this attribute.